content/source/docs/enterprise/api/oauth-clients.html.md

```diff
@@ -25,10 +153,12 @@ This endpoint allows you to create a VCS connection between an organization and
 * [Bitbucket Cloud](https://confluence.atlassian.com/bitbucket/app-passwords-828781300.html)
 * [GitLab, GitLab Community Edition, and GitLab Enterprise Edition](https://docs.gitlab.com/ce/user/profile/personal_access_tokens.html#creating-a-personal-access-token)
 
-Result  | Status and response
---------|------------------------
-Success | HTTP 201 and a JSON API document (`type: "oauth-clients"`).
-Failure | HTTP 4XX and a JSON API list of error messages.
+Status  | Response                                        | Reason
+--------|-------------------------------------------------|----------
+[201][] | [JSON API document][] (`type: "oauth-clients"`) | OAuth Client successfully created
+[404][] | [JSON API error object][]                       | OAuth Client not found or user unauthorized to perform action
```

Comment:
"not found" doesn't seem to make sense for a create request, right? So it's probably only the unauthorized case.

Reply:
If the organization is not found the response will be 404. Should this be corrected to "Organization"?

Reply:
The label has been updated.